src/custom/components/swap/EthFlow/helpers.ts

```diff
@@ -311,10 +311,13 @@ export function _getCurrencyForVisualiser<T>(native: T, wrapped: T, isWrap: bool
   }
 }
 
-export type ActionButtonParams = Pick<Props, 'isNativeIn'> &
-  Pick<DerivedEthFlowStateProps, 'approveError' | 'wrapError' | 'approveState' | 'wrapState' | 'isExpertMode'> &
+export type ActionButtonParams = Pick<
```

Comment:
Its not coming from this PR, but this type definition is crazy. It should be simpler, maybe defining some intermediate types

Reply:
Agree, we should refactor EthFlow next